docs/2-protocol-overview/fees.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebar_position: 4
+---
+
+# Fees
+
+:::caution
+
+This article may contain temporary information and applies only to the ErgoDEX V1
+
+:::
+
+ErgoDEX protocol V1 includes 3 types of fees:
+
+| Fee              | Assignment                         | Unit                              | Amount                            |
+| ---------------- | ---------------------------------- | --------------------------------- | --------------------------------- |
+| Network Fee      | Paid to Miners or Node stake pools | Native network currency (ERG/ADA) | min - ∞ (user choice)             |
```

Comment:
Should this be miner fees as stated above? Or should it be changed everywhere to either miner fee or network fee. Maybe network fee makes more sense as it can be consistent with cardano ErgoDEX.